de/translation.json

```diff
@@ -271,13 +269,13 @@
   "Common.Cancel": "Abbrechen",
   "Common.Challenge": "Herausforderung",
   "Common.Challenge_plural": "Herausforderungen",
-  "Common.Character": "🦘 Character",
+  "Common.Character": "Charakter",
   "Common.CharacterClass": "Charakter-Klasse",
-  "Common.CharacterLight": "🦘 Character Light",
-  "Common.CharacterSpecific": "🦘 As is with most Braytech features, this view is character-specific.",
+  "Common.CharacterLight": "Charakter Licht",
+  "Common.CharacterSpecific": "Wie die meisten Braytech Funktionen, ist diese Ansicht charakterspezifisch.",
```

Comment:
"Braytech-Funktionen" with a hyphen. Additionally, the comma is superfluous and should be removed.